packages/fxa-settings/src/components/Settings/FlowRecoveryKeyDownload/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { logViewEvent } from '../../../lib/metrics';
+
+export type FlowRecoveryKeyDownloadProps = {
+  localizedBackButtonTitle: string;
+  localizedPageTitle: string;
+  navigateBackward: () => void;
+  navigateForward: () => void;
+  recoveryKeyValue: string;
+  viewName: string;
+};
+
+export const FlowRecoveryKeyDownload = ({
+  localizedBackButtonTitle,
+  localizedPageTitle,
+  navigateBackward,
+  navigateForward,
+  recoveryKeyValue,
+  viewName,
+}: FlowRecoveryKeyDownloadProps) => {
+  return (
+    <FlowContainer
+      title={localizedPageTitle}
+      onBackButtonClick={navigateBackward}
+      {...{ localizedBackButtonTitle }}
+    >
+      <div className="w-full flex flex-col gap-4">
+        <ProgressBar currentStep={3} numberOfSteps={4} />
+        <RecoveryKeyImage className="my-6 mx-auto" />
+
+        <FtlMsg id="flow-recovery-key-download-heading">
+          <h2 className="font-bold text-xl">
+            {/* This is an em dash - add space? */}
+            Account recovery key generated — store it in a place you’ll remember
+          </h2>
+        </FtlMsg>
+        <FtlMsg id="flow-recovery-key-download-info">
+          <p className="text-sm">
+            This key will help recover your data if you forget your password.
+          </p>
+        </FtlMsg>
+        {/* TODO Add copy metric flow.settings.account-recovery.copy-option */}
+        <DataBlock value={recoveryKeyValue} isInline />
+        <FtlMsg id="flow-recovery-key-download-storage-ideas-heading">
+          <h3 className="font-bold -mb-4">
+            Some ideas for storing your recovery key
+          </h3>
+        </FtlMsg>
+        <ul>
+          <FolderIconListItem>
+            <FtlMsg id="flow-recovery-key-download-storage-ideas-folder">
+              <p>Memorable folder in your device</p>
+            </FtlMsg>
+          </FolderIconListItem>
+          <GlobeIconListItem>
+            <FtlMsg id="flow-recovery-key-download-storage-ideas-cloud">
+              <p>Trusted cloud storage</p>
+            </FtlMsg>
+          </GlobeIconListItem>
+          <PrinterIconListItem>
+            <FtlMsg id="flow-recovery-key-download-storage-ideas-print">
+              <p>Print and keep a physical copy</p>
+            </FtlMsg>
+          </PrinterIconListItem>
+          <LockIconListItem>
+            <FtlMsg id="flow-recovery-key-download-storage-ideas-pwd-manager">
+              <p>Password manager</p>
+            </FtlMsg>
+          </LockIconListItem>
+        </ul>
+        <ButtonDownloadRecoveryKey
+          {...{ navigateForward, recoveryKeyValue, viewName }}
+        />
+        <FtlMsg id="flow-recovery-key-download-next-link">
+          <Link
+            to=""
+            className="text-sm link-blue text-center py-2"
+            onClick={() => {
+              logViewEvent(`flow.${viewName}`, 'skip-download');
+              navigateForward();
+            }}
+          >
+            Next
+          </Link>
```

Comment:
This link takes up the entire width:

![image](https://github.com/mozilla/fxa/assets/13018240/449f633e-fd1b-43ce-beeb-6e46197c3828)